dev-scripts/setup-service.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# For dev mode, create a dummy service pointing to the Rancher instance running on localhost.
+# Allows the project resources kubernetes extension to work.
+# Not needed for CI, helm or docker setups because the service is automatically created.
+ip=$(ip addr show docker0 | awk '/inet /{gsub(/\/16/, ""); print $2}')
+
+cat <<EOF | sed -e "s/{ip}/${ip}/" | kubectl apply -f -
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: port-forward
+  name: port-forward
+  namespace: cattle-system
+spec:
+  containers:
+  - env:
+    - name: REMOTE_HOST
+      value: {ip} # localhost gateway
+    - name: REMOTE_PORT
+      value: "8443"
+    - name: LOCAL_PORT
+      value: "443"
+    image: marcnuri/port-forward
```

Comment:
This image is hosted outside of the rancher org, hasn't been updated in ~2 years, and isn't pinned to a specific version. Would there be an alternative image that we can use that has a lower risk of takeover/malicious usage?

Reply:
That's part of why I kept this in a separate script instead of trying to automate it in rancher. I'll look for an alternative, or if you know of one I'll happily swap it out.